.prettierrc

```diff
@@ -2,6 +2,5 @@
   "singleQuote": true,
   "trailingComma": "all",
   "bracketSpacing": false,
-  "printWidth": 100,
-  "parser": "typescript"
```

Comment:
Here, and other places, I removed `parser`. It prevents Prettier from formatting other files which it’s pretty good at